unitq/README.md

```diff
@@ -0,0 +1,39 @@
+# unitQ
+
+## Overview
+
+The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog.By delivering unitQ metrics to Datadog, users can leverage Datadog's graphing and alerting capabilities to track user feedback ensuring a happy customer base
+
+## Setup
+
+### Configuration
+
+1. In unitQ, go to **Integrations**.
```

Comment:
A hyperlink to the Integrations page within unitQ would be helpful here.

Reply:
Unfortunately, we can't do that at the moment.